control-plane/roles/monitoring/templates/grafana_dashboards/shoot-cluster-status.yaml

```diff
@@ -0,0 +1,1112 @@
+---
```

Comment:
Not necessary, do not care about Gardener related stuff for now.

Reply:
removed